doc/source/ray-air/predictors.rst

```diff
@@ -104,10 +104,39 @@ To configure the computation resources for your `BatchPredictor`, you have to se
 
 Here are some examples:
 
+**1. Use CPUs for Batch prediction:**
+
+- Scale from 1 to 4 workers, depending on the dataset size.
+
+- Each worker has 3 CPUs for inference.
+
+.. literalinclude:: doc_code/predictors.py
+    :language: python
+    :start-after: __configure_batch_predictor_1_start__
+    :end-before: __configure_batch_predictor_1_end__
```

Comment:
Can we keep these sections isolated?

One section only shows using multiple CPUs. One section only shows using GPUs. One section only shows configuring actor scaling.

Reply:
Agree. I've changed that!